tests/test_algorithm/test_id_alg.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
```

Comment:
Don't forget there are example graphs available in https://github.com/y0-causal-inference/y0/blob/main/src/y0/examples.py

Reply:
I didn't know about this!  I will add the examples from   this paper:
```
      Shpitser, I., & Pearl, J. (2008). Complete Identification Methods for the Causal Hierarchy. Journal of Machine Learning Research. 
```